### ISSUE TITLE
Error when running tflint - plugin process exited

### ISSUE THREAD

**Issue by joshghent** [terraform-linters/tflint-ruleset-aws#866]
Hey there,

First of all, thanks a lot for your hard work on this project!

Now to the actual issue.
I've not done any investigation as to the replication steps for what is triggering this error but here is the error in it's totality that for a particular repo (private unfortunately) causes an error for all machines I have tried this one (3 so far).

The crux of the issue is that although tflint runs, it doesn't actually seem to do any linting. I even added a deliberate issue (invalid ec2 instance type) and found that tflint didn't pick it up. 
When looking into the debug logs, it appears as though it's silently failing.

```bash
> docker run --rm -v $(pwd):/data -t --entrypoint /bin/sh ghcr.io/terraform-linters/tflint -c "tflint --init && TFLINT_LOG=debug tflint"
Installing "aws" plugin...
Installed "aws" (source: github.com/terraform-linters/tflint-ruleset-aws, version: 0.33.0)
12:52:15 config.go:177: [INFO] Load config: .tflint.hcl
12:52:15 config.go:334: [DEBUG] Config loaded
12:52:15 config.go:335: [DEBUG]   CallModuleType: local
12:52:15 config.go:336: [DEBUG]   CallModuleTypeSet: false
12:52:15 config.go:337: [DEBUG]   Force: false
12:52:15 config.go:338: [DEBUG]   ForceSet: false
12:52:15 config.go:339: [DEBUG]   DisabledByDefault: false
12:52:15 config.go:340: [DEBUG]   DisabledByDefaultSet: false
12:52:15 config.go:341: [DEBUG]   PluginDir:
12:52:15 config.go:342: [DEBUG]   PluginDirSet: false
12:52:15 config.go:343: [DEBUG]   Format:
12:52:15 config.go:344: [DEBUG]   FormatSet: false
12:52:15 config.go:345: [DEBUG]   Varfiles:
12:52:15 config.go:346: [DEBUG]   Variables:
12:52:15 config.go:347: [DEBUG]   Only:
12:52:15 config.go:348: [DEBUG]   IgnoreModules:
12:52:15 config.go:352: [DEBUG]   Rules:
12:52:15 config.go:354: [DEBUG]     terraform_documented_variables: true
12:52:15 config.go:354: [DEBUG]     terraform_typed_variables: true
12:52:15 config.go:354: [DEBUG]     terraform_required_version: false
12:52:15 config.go:354: [DEBUG]     aws_resource_missing_tags: false
12:52:15 config.go:354: [DEBUG]     terraform_naming_convention: true
12:52:15 config.go:354: [DEBUG]     terraform_deprecated_interpolation: true
12:52:15 config.go:354: [DEBUG]     terraform_required_providers: true
12:52:15 config.go:354: [DEBUG]     terraform_unused_required_providers: true
12:52:15 config.go:354: [DEBUG]     terraform_documented_outputs: true
12:52:15 config.go:356: [DEBUG]   Plugins:
12:52:15 config.go:358: [DEBUG]     aws: enabled=true, version=0.33.0, source=github.com/terraform-linters/tflint-ruleset-aws
12:52:15 config.go:463: [INFO] The "terraform" plugin block is not found. Enable the plugin "terraform" automatically
12:52:15 option.go:77: [DEBUG] CLI Options
12:52:15 option.go:78: [DEBUG]   CallModuleType: local
12:52:15 option.go:79: [DEBUG]   Force: false
12:52:15 option.go:80: [DEBUG]   Format:
12:52:15 option.go:81: [DEBUG]   Varfiles:
12:52:15 option.go:82: [DEBUG]   Variables:
12:52:15 option.go:83: [DEBUG]   EnableRules:
12:52:15 option.go:84: [DEBUG]   DisableRules:
12:52:15 option.go:85: [DEBUG]   Only:
12:52:15 option.go:86: [DEBUG]   EnablePlugins:
12:52:15 option.go:87: [DEBUG]   IgnoreModules:
12:52:15 loader.go:39: [INFO] Initialize new loader
12:52:15 loader.go:81: [INFO] Building the root module while calling local child modules...
12:52:15 runner.go:46: [INFO] Initialize new runner for root
12:52:15 discovery.go:90: [DEBUG] Find plugin path: /root/.tflint.d/plugins/github.com/terraform-linters/tflint-ruleset-aws/0.33.0/tflint-ruleset-aws
12:52:15 discovery.go:54: [INFO] Plugin "aws" found
12:52:15 [DEBUG] cmdrunner/cmd_runner.go:73: starting plugin: path=/root/.tflint.d/plugins/github.com/terraform-linters/tflint-ruleset-aws/0.33.0/tflint-ruleset-aws args=["/root/.tflint.d/plugins/github.com/terraform-linters/tflint-ruleset-aws/0.33.0/tflint-ruleset-aws"]
12:52:15 [DEBUG] cmdrunner/cmd_runner.go:80: plugin started: path=/root/.tflint.d/plugins/github.com/terraform-linters/tflint-ruleset-aws/0.33.0/tflint-ruleset-aws pid=18
12:52:15 [DEBUG] go-plugin@v1.6.3/client.go:827: waiting for RPC address: plugin=/root/.tflint.d/plugins/github.com/terraform-linters/tflint-ruleset-aws/0.33.0/tflint-ruleset-aws
12:52:15 [DEBUG] go-plugin@v1.6.3/client.go:880: using plugin: version=11
12:52:15 [DEBUG] go-plugin@v1.6.3/client.go:1216: tflint-ruleset-aws: 12:52:15 [DEBUG] go-plugin@v1.6.1/server.go:419: plugin address: network=unix address=/tmp/plugin3025087799
12:52:15 discovery.go:33: [INFO] Plugin "terraform" is not installed, but the bundled plugin is available.
12:52:15 discovery.go:54: [INFO] Plugin "terraform" found
12:52:15 [DEBUG] cmdrunner/cmd_runner.go:73: starting plugin: path=/usr/local/bin/tflint args=["/usr/local/bin/tflint", "--act-as-bundled-plugin"]
12:52:15 [DEBUG] cmdrunner/cmd_runner.go:80: plugin started: path=/usr/local/bin/tflint pid=25
12:52:15 [DEBUG] go-plugin@v1.6.3/client.go:827: waiting for RPC address: plugin=/usr/local/bin/tflint
12:52:16 [DEBUG] go-plugin@v1.6.3/client.go:1216: tflint: 12:52:16 [DEBUG] go-plugin@v1.6.3/server.go:419: plugin address: network=unix address=/tmp/plugin823572343
12:52:16 [DEBUG] go-plugin@v1.6.3/client.go:880: using plugin: version=11
12:52:16 [DEBUG] host2plugin/client.go:124: starting host-side gRPC server
12:52:16 [DEBUG] host2plugin/client.go:124: starting host-side gRPC server
12:52:16 [DEBUG] go-plugin@v1.6.3/grpc_stdio.go:142: stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
12:52:16 [INFO]  go-plugin@v1.6.3/client.go:780: plugin process exited: plugin=/usr/local/bin/tflint id=25
12:52:16 [DEBUG] go-plugin@v1.6.3/client.go:558: plugin exited
12:52:16 [DEBUG] go-plugin@v1.6.3/grpc_stdio.go:142: stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
12:52:16 [INFO]  go-plugin@v1.6.3/client.go:780: plugin process exited: plugin=/root/.tflint.d/plugins/github.com/terraform-linters/tflint-ruleset-aws/0.33.0/tflint-ruleset-aws id=18
12:52:16 [DEBUG] go-plugin@v1.6.3/client.go:558: plugin exited
```

I also got this similar looking error in another repository
```
All plugins are already installed
12:58:07 config.go:177: [INFO] Load config: .tflint.hcl
12:58:07 config.go:185: [INFO] file not found
12:58:07 config.go:192: [INFO] Load config: /root/.tflint.hcl
12:58:07 config.go:200: [INFO] file not found
12:58:07 config.go:203: [INFO] Use default config
12:58:07 config.go:463: [INFO] The "terraform" plugin block is not found. Enable the plugin "terraform" automatically
12:58:07 option.go:77: [DEBUG] CLI Options
12:58:07 option.go:78: [DEBUG]   CallModuleType: local
12:58:07 option.go:79: [DEBUG]   Force: false
12:58:07 option.go:80: [DEBUG]   Format:
12:58:07 option.go:81: [DEBUG]   Varfiles:
12:58:07 option.go:82: [DEBUG]   Variables:
12:58:07 option.go:83: [DEBUG]   EnableRules:
12:58:07 option.go:84: [DEBUG]   DisableRules:
12:58:07 option.go:85: [DEBUG]   Only:
12:58:07 option.go:86: [DEBUG]   EnablePlugins:
12:58:07 option.go:87: [DEBUG]   IgnoreModules:
12:58:07 loader.go:39: [INFO] Initialize new loader
12:58:07 loader.go:81: [INFO] Building the root module while calling local child modules...
12:58:07 runner.go:46: [INFO] Initialize new runner for root
12:58:07 discovery.go:33: [INFO] Plugin "terraform" is not installed, but the bundled plugin is available.
12:58:07 discovery.go:54: [INFO] Plugin "terraform" found
12:58:07 [DEBUG] cmdrunner/cmd_runner.go:73: starting plugin: path=/usr/local/bin/tflint args=["/usr/local/bin/tflint", "--act-as-bundled-plugin"]
12:58:07 [DEBUG] cmdrunner/cmd_runner.go:80: plugin started: path=/usr/local/bin/tflint pid=15
12:58:07 [DEBUG] go-plugin@v1.6.3/client.go:827: waiting for RPC address: plugin=/usr/local/bin/tflint
12:58:07 [DEBUG] go-plugin@v1.6.3/client.go:1216: tflint: 12:58:07 [DEBUG] go-plugin@v1.6.3/server.go:419: plugin address: network=unix address=/tmp/plugin3159098702
12:58:07 [DEBUG] go-plugin@v1.6.3/client.go:880: using plugin: version=11
12:58:07 [DEBUG] host2plugin/client.go:124: starting host-side gRPC server
12:58:07 [DEBUG] go-plugin@v1.6.3/grpc_stdio.go:142: stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
12:58:07 [INFO]  go-plugin@v1.6.3/client.go:780: plugin process exited: plugin=/usr/local/bin/tflint id=15
12:58:07 [DEBUG] go-plugin@v1.6.3/client.go:558: plugin exited
```

This occurs on both the docker run time but also when installed locally via brew.

This was done with
```
TFLint version 0.56.0
+ ruleset.aws (0.33.0)
+ ruleset.terraform (0.11.0-bundled)
```

Hope that's enough information. Let me know if you want me to look into anything else.
